cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/Conversion.java

```diff
@@ -306,6 +311,34 @@ private void addCgmesSvMetadata(Network network) {
         }
     }
 
+    private void addCgmesSshMetadata(Network network) {
+        PropertyBags sshDescription = cgmes.fullModel(CgmesSubset.STEADY_STATE_HYPOTHESIS.getProfile());
+        if (sshDescription != null && !sshDescription.isEmpty()) {
+            CgmesSshMetadataAdder adder = network.newExtension(CgmesSshMetadataAdder.class)
+                    .setDescription(sshDescription.get(0).getId("description"))
+                    .setSshVersion(sshDescription.get(0).asInt("version"))
+                    .setModelingAuthoritySet(sshDescription.get(0).getId("modelingAuthoritySet"));
+            sshDescription.pluckLocals("DependentOn").forEach(adder::addDependency);
+            adder.add();
+        }
+    }
+
+    private void addCgmesSshControlAreas(Network network) {
+        PropertyBags sshControlAreas = cgmes.controlAreas();
+        if (sshControlAreas != null && !sshControlAreas.isEmpty()) {
+            CgmesSshControlAreasAdder adder = network.newExtension(CgmesSshControlAreasAdder.class);
+
+            sshControlAreas.forEach(sshControlArea -> {
+                String id = sshControlArea.getId("ControlArea");
+                double netInterchange = sshControlArea.asDouble("netInterchange");
+                double pTolerance = sshControlArea.asDouble("pTolerance");
+                ControlArea controlArea = new ControlArea(id, netInterchange, pTolerance);
+                adder.addControlArea(controlArea);
+            });
+            adder.add();
+        }
+    }
```

Comment:
To confirm, but maybe it might be a good idea to use the extension in `powsybl-balance-adjustements `. I will look into it.

Reply:
We will analyse the `powsybl-balance-adjustements` extension for a later refactoring.